### ISSUE TITLE
Multiverse-Portals Not sending to the correct world sometimes

### ISSUE THREAD

**Issue by fahlur** [Multiverse/Multiverse-Portals#463]
We have regular world resets every 2 weeks of a specific world. And lately for some reason the portal to our "mining world" will just pick this one location in our main world and teleport you there instead of the actual mining world. This is a fairly new issue for us,. but after a server restart it seems to "fix itself" but only after a restart. This never use to happen. So no idea on this..

**Comment by Jadiction** [comment]
Closing ticket because it's years old. Update to the latest versions of Multiverse (Core, Portals and any others you may use). Please open a new ticket if this issue persists.
